### ISSUE TITLE
Activity Monitoring: empty response from backend

### ISSUE THREAD

**Issue by dsimansk** [syndesisio/syndesis#2755]
## This is a...
<!-- Check ONLY one of the following options with "x" -->
<pre><code>
[ ] Feature request
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>


## The problem
I've tried to rerun the soak test scenario on today's tag `1.4.1-20180605`. However, I can't see any Activity displayed on a page. Backend's response is empty, even for the straight request to server.

## Expected behavior
Activities over time are visibile

## Screenshot
<img width="981" alt="screen shot 2018-06-05 at 16 01 46" src="https://user-images.githubusercontent.com/5637792/40980873-c9bce51c-68d9-11e8-98ab-a624a7b6d36b.png">


## Request and Response Data
```
GET /api/v1/activity/integrations/i-LEFAjpOhQlZNq3ZGF-7z HTTP/1.1
Host: syndesis.apps.perf2.xpaas
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
Accept: application/json, text/plain, */*
SYNDESIS-XSRF-TOKEN: awesome
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.183 Safari/537.36 Vivaldi/1.96.1147.42
DNT: 1
Referer: https://syndesis.apps.perf2.xpaas/integrations/i-LEFAjpOhQlZNq3ZGF-7z?view=activity
Accept-Encoding: gzip, deflate, br
Accept-Language: en-GB,en-US;q=0.9,en;q=0.8
Cookie: b2a2e2613ad00502bbe3dae4fed6f03a=46f39228ecc2cd9481e1508468b70f1a; _oauth_proxy=eHBhYXNxZUBjbHVzdGVyLmxvY2FsfGlyaXFEdW90UUNBdExzYk5kME5DR3ZiQk5rcnlwdlozRldLMHhDNk5EZDJKMkpxdmovelB1QXo1V3dsMTQwbGNLQU1NanNZTFpkY3ZBWm89fC02MjEzNTU5NjgwMHw=|1528204485|BBjMAmtlhA04gLmf1bEnV6g5hCk=
```
Response
```
HTTP/1.1 200 OK
Cache-Control: no-cache, no-store, max-age=0, must-revalidate, proxy-revalidate, s-maxage=0
Content-Length: 2
Content-Type: application/json
Date: Tue, 05 Jun 2018 14:03:07 GMT
Expires: 0
Gap-Auth: xpaasqe@cluster.local
Gap-Upstream-Address: syndesis-server
Pragma: no-cache
Strict-Transport-Security: max-age=31536000 ; includeSubDomains
Syndesis-Xsrf-Token: awesome
X-Application-Context: application
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Xss-Protection: 1; mode=block
```


```
syndesis=# select * from pg_stat_activity;
 datid | datname  |  pid  | usesysid | usename  | application_name | client_addr | client_hostname | client_port |         backend_start         |          xact_start           |          query_start          |         state_change          | waiting |        state
   | backend_xid | backend_xmin |                                                                                                                                                                                                                     query

-------+----------+-------+----------+----------+------------------+-------------+-----------------+-------------+-------------------------------+-------------------------------+-------------------------------+-------------------------------+---------+------------------
---+-------------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 16385 | syndesis |  1819 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39722 | 2018-06-05 13:34:32.642486+00 | 2018-06-05 13:54:52.01888+00  | 2018-06-05 13:54:52.019304+00 | 2018-06-05 13:54:52.021046+00 | f       | idle in transacti
on |             |         1082 | select path,value,ovalue from jsondb where path LIKE $1 and path >= $2 and path < $3 order by path ASC
 16385 | syndesis |  1820 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39724 | 2018-06-05 13:34:32.70542+00  |                               | 2018-06-05 13:48:51.312603+00 | 2018-06-05 13:48:51.312611+00 | f       | idle
   |             |              | COMMIT
 16385 | syndesis |  1821 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39726 | 2018-06-05 13:34:32.712346+00 | 2018-06-05 13:47:59.174423+00 | 2018-06-05 13:47:59.174448+00 | 2018-06-05 13:47:59.175183+00 | f       | idle in transacti
on |             |         1015 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1822 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39728 | 2018-06-05 13:34:32.719077+00 | 2018-06-05 13:44:33.068511+00 | 2018-06-05 13:44:33.068636+00 | 2018-06-05 13:44:33.069414+00 | f       | idle in transacti
on |             |          938 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1823 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39730 | 2018-06-05 13:34:32.725039+00 | 2018-06-11 05:54:29.199144+00 | 2018-06-11 05:54:29.199516+00 | 2018-06-11 05:54:29.200688+00 | f       | idle in transacti
on |             |        73863 | select path,value,ovalue from jsondb where path LIKE $1 and path >= $2 and path < $3 order by path ASC
 16385 | syndesis |  1824 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39732 | 2018-06-05 13:34:32.731351+00 | 2018-06-05 13:42:16.933953+00 | 2018-06-05 13:42:16.934602+00 | 2018-06-05 13:42:16.936566+00 | f       | idle in transacti
on |             |          907 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1825 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39734 | 2018-06-05 13:34:32.737416+00 | 2018-06-05 13:35:23.349052+00 | 2018-06-05 13:35:23.349185+00 | 2018-06-05 13:35:23.357102+00 | f       | idle in transacti
on |             |          818 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1826 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39736 | 2018-06-05 13:34:32.743253+00 | 2018-06-05 13:43:24.64441+00  | 2018-06-05 13:43:24.644435+00 | 2018-06-05 13:43:24.645302+00 | f       | idle in transacti
on |             |          920 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1827 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39738 | 2018-06-05 13:34:32.749213+00 | 2018-06-05 13:45:41.182523+00 | 2018-06-05 13:45:41.182548+00 | 2018-06-05 13:45:41.183351+00 | f       | idle in transacti
on |             |          958 | SELECT data FROM filestore WHERE path=$1
 16385 | syndesis |  1828 |    16384 | syndesis |                  | 10.129.0.1  |                 |       39740 | 2018-06-05 13:34:32.755448+00 | 2018-06-05 13:34:43.451718+00 | 2018-06-05 13:34:43.462198+00 | 2018-06-05 13:34:43.484651+00 | f       | idle in transacti
on |             |          805 | SELECT p.proname,p.oid  FROM pg_catalog.pg_proc p, pg_catalog.pg_namespace n  WHERE p.pronamespace=n.oid AND n.nspname='pg_catalog' AND ( proname = 'lo_open' or proname = 'lo_close' or proname = 'lo_creat' or proname = 'lo_unlink' or pr
oname = 'lo_lseek' or proname = 'lo_lseek64' or proname = 'lo_tell' or proname = 'lo_tell64' or proname = 'loread' or proname = 'lowrite' or proname = 'lo_truncate' or proname = 'lo_truncate64')
 16385 | syndesis |  2679 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41190 | 2018-06-05 13:46:50.462334+00 | 2018-06-05 13:46:51.067104+00 | 2018-06-05 13:46:51.070437+00 | 2018-06-05 13:46:51.071236+00 | f       | idle in transacti
on |             |          989 | SELECT p.proname,p.oid  FROM pg_catalog.pg_proc p, pg_catalog.pg_namespace n  WHERE p.pronamespace=n.oid AND n.nspname='pg_catalog' AND ( proname = 'lo_open' or proname = 'lo_close' or proname = 'lo_creat' or proname = 'lo_unlink' or pr
oname = 'lo_lseek' or proname = 'lo_lseek64' or proname = 'lo_tell' or proname = 'lo_tell64' or proname = 'loread' or proname = 'lowrite' or proname = 'lo_truncate' or proname = 'lo_truncate64')
 16385 | syndesis |  2749 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41448 | 2018-06-05 13:47:53.57209+00  | 2018-06-05 13:48:51.226192+00 | 2018-06-05 13:48:51.226505+00 | 2018-06-05 13:48:51.232158+00 | f       | idle in transacti
on |             |         1027 | select path,value,ovalue from jsondb where path LIKE $1 order by path ASC
 16385 | syndesis |  2750 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41450 | 2018-06-05 13:47:53.575772+00 | 2018-06-11 06:00:31.250189+00 | 2018-06-11 06:00:31.250458+00 | 2018-06-11 06:00:31.250471+00 | f       | idle in transacti
on |             |        73917 | select path,value,ovalue from jsondb where path LIKE $1 order by path ASC
 16385 | syndesis |  2820 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41568 | 2018-06-05 13:48:51.245227+00 |                               | 2018-06-05 13:48:51.338238+00 | 2018-06-05 13:48:51.338246+00 | f       | idle
   |             |              | COMMIT
 16385 | syndesis |  2821 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41570 | 2018-06-05 13:48:51.24925+00  |                               | 2018-06-05 13:48:51.332999+00 | 2018-06-05 13:48:51.334084+00 | f       | idle
   |             |              | select path,value,ovalue from jsondb where path LIKE $1 and path >= $2 and path < $3 order by path ASC
 16385 | syndesis |  2823 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41574 | 2018-06-05 13:48:51.250309+00 |                               | 2018-06-05 13:50:50.657709+00 | 2018-06-05 13:50:50.659322+00 | f       | idle
   |             |              | select path,value,ovalue from jsondb where path LIKE $1 and path >= $2 and path < $3 order by path ASC
 16385 | syndesis |  2822 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41572 | 2018-06-05 13:48:51.250937+00 | 2018-06-11 06:00:06.158511+00 | 2018-06-11 06:00:06.164551+00 | 2018-06-11 06:00:06.164563+00 | f       | idle in transacti
on |             |        73911 | select path,value,ovalue from jsondb where path LIKE $1 order by path ASC
 16385 | syndesis |  2824 |    16384 | syndesis |                  | 10.129.0.1  |                 |       41576 | 2018-06-05 13:48:51.253989+00 |                               | 2018-06-05 13:48:51.335479+00 | 2018-06-05 13:48:51.337027+00 | f       | idle
   |             |              | select path,value,ovalue from jsondb where path LIKE $1 and path >= $2 and path < $3 order by path ASC
 16385 | syndesis | 44760 |       10 | postgres | psql             |             |                 |          -1 | 2018-06-11 05:59:47.419084+00 | 2018-06-11 06:00:31.38028+00  | 2018-06-11 06:00:31.38028+00  | 2018-06-11 06:00:31.380286+00 | f       | active
   |             |        73917 | select * from pg_stat_activity;
 16385 | syndesis | 44795 |       10 | postgres |                  |             |                 |             | 2018-06-11 06:00:24.603251+00 | 2018-06-11 06:00:24.630976+00 | 2018-06-11 06:00:24.630976+00 | 2018-06-11 06:00:24.630978+00 | f       | active
   |             |        73917 | autovacuum: VACUUM public.jsondb
(20 rows)
```

## Tasks involved / Steps to Reproduce
<!--
Enlist all the acceptance criteria for new features or the steps required to reproduce the bug/regression reported.
-->
1. Deploy integration
2. Navigate to activity tracking

**Comment by dsimansk** [comment]
Cc @tplevko @chirino

**Comment by dsimansk** [comment]
@chirino any idea what can cause be causing this issue? I really can't continue with any relevant perf numbers without Activities being written into DB.

**Comment by zregvart** [comment]
I think this might have been addressed with #2775

**Comment by chirino** [comment]
Yep. Retry a new nightly.

**Comment by dsimansk** [comment]
Fixed by #2775.
